manual/library/builtin.etex

```diff
@@ -190,7 +192,10 @@ exception Failure of string
 \index{Failure@\verb`Failure`}
 \begin{ocamldocdescription}
    Exception raised by library functions to signal that they are
-   undefined on the given arguments.
+   undefined on the given arguments.  The string gives some information
+   to the programmer; it is \emph{bad practice} to pattern match
```

Comment:
Why bad practice ? Don't even give the reader the idea that this is acceptable. It *must not* bet patterned matched. Also it should be said here, that nowadays using `Failure` should be frowned upon and either `result` or `option` should be used instead.